seqr/views/utils/dataset_utils.py

```diff
@@ -366,44 +350,59 @@ def _load_rna_seq_file(
     missing_required_fields = defaultdict(list)
     gene_ids = set()
     current_sample = None
-    for line in tqdm(f, unit=' rows'):
-        row = dict(zip(header, _parse_tsv_row(line)))
-        for sample_id, row_dict in _parse_rna_row(
```

Comment:
`_parse_rna_row` is a generator so this was executing a `for` loop over it, but thats confusing and implies that there can be multiple sample/row_dicts per row. In fact, there are either 0 or 1 valid row_dicts per row, so it makes more sense to remove this helper and the unnecessary level of nesting it introduces